standalone/src/models/order/line-item-return-item/index.ts

```diff
@@ -1,7 +1,26 @@
-import LegacyBuilder from './builder';
-import * as constants from './constants';
+import {
+  RestModelBuilder,
+  GraphqlModelBuilder,
+  CompatLineItemReturnItemModelBuilder,
+} from './builders';
+import * as modelPresets from './presets';
 
+export * as constants from './constants';
```

Comment:
Constants should be exported as part of each model ([example](https://github.com/commercetools/test-data/blob/main/standalone/src/models/product-type/attribute-definition/index.ts)).

Reply:
We will also need to update the constants to export the TS enum so we don't have TS typing issues with the GraphQL model.
[Here's a reference](https://github.com/commercetools/test-data/pull/864#discussion_r2150321232) of what we've done in another model.

![image](https://github.com/user-attachments/assets/e3b84b70-b575-4662-9a8f-157d85e1d930)

![image](https://github.com/user-attachments/assets/fd983f46-9102-4667-9418-e95f6b74ba51)